service/scaladsl/integration-tests/src/test/scala/com/lightbend/lagom/scaladsl/it/mocks/MockService.scala

```diff
@@ -146,15 +146,15 @@ class MockServiceImpl(implicit mat: Materializer, ec: ExecutionContext) extends
     Future.successful(stream.map(req => MockResponseEntity(1, req)))
   }
 
-  override def customHeaders = HeaderServiceCall { (requestHeader, headerName) =>
+  override def customHeaders = ServerServiceCall { (requestHeader, headerName) =>
     val headerValue = requestHeader.getHeader(headerName).getOrElse {
       throw NotFound("Header " + headerName)
     }
-    Future.successful((ResponseHeader.OK.withStatus(201).withHeader("Header-Name", headerName), headerValue))
+    Future.successful((ResponseHeader.Ok.withStatus(201).withHeader("Header-Name", headerName), headerValue))
   }
```

Comment:
Oh, I see. `ResponseHeader.Ok` is not only for the Status `OK` but as an umbrella for `2xx` responses.